snuba/datasets/entities/generic_metrics.py

```diff
@@ -97,7 +103,14 @@ def __init__(
         )
 
     def get_query_processors(self) -> Sequence[QueryProcessor]:
-        return [TagsTypeTransformer()]
+        return [
+            TagsTypeTransformer(),
+            MappedGranularityProcessor(
+                accepted_granularities=PERFORMANCE_GRANULARITIES,
+                default_granularity_enum=DEFAULT_MAPPED_GRANULARITY_ENUM,
+            ),
+            TimeSeriesProcessor({"bucketed_time": "timestamp"}, ("timestamp",)),
```

Comment:
Do you need this processor if you have the granularity processor above?

Reply:
yes, because the product wants to round timestamps to odd intervals like 15m or 3h or whatever which isn't handled by the granularity processor alone